tutorials/cookbook/Recipes Thomas.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print(H_enron_cleaned

or just don't defined the new variable, I thin cleanup() works "in place"
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>